.github/workflows/ComplianceScanning.yml

```diff
@@ -0,0 +1,68 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "ComplianceScanning"
+
+on:
+  push:
```

Comment:
Should there be a path constraint here as well?

Reply:
Technically the whole repo should have already had this before we even started. So everything needs scanned.

Reply:
Makes sense!